java/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for HMCTS Java Microservices
 name: java
-version: 2.16.1
+version: 2.16.2
```

should probably be 2.17.0
```suggestion
version: 2.17.0
```